app/assets/javascripts/components/pageElements/content_container.js.jsx

```diff
@@ -0,0 +1,58 @@
+var ContentContainer = React.createClass({
+  getInitialState: function(){
+    return {films: [], selectedFilm: ''}
```

Comment:
Not necessary to save `selectedFilm` in `state` since it's never used again. Less state is better. More is less.